libevm/VMFace.h

```diff
@@ -25,14 +25,30 @@ namespace dev
 namespace eth
 {
 
-struct VMException: virtual Exception {};
-#define ETH_SIMPLE_EXCEPTION_VM(X) struct X: virtual VMException { const char* what() const noexcept override { return #X; } }
+struct VMException: Exception {};
```

Comment:
Any reason why these were virtually inherited? It's not like they are going to be inherited several times here in libevm and `VMException` and children don't have data members anyway!

I'm changing it because gcc was issuing a warning for `RevertInstruction` move assignment, which is unsafe with virtual base (virtual base can be move-assigned several times if inherited several times)